README.md

```diff
@@ -21,27 +21,35 @@ Note: This starts a php server on port 3000.
 
 ## Database Setup
 
+#### Development setup with test data
 Login to database via Mariadb
 
 Once logged on, do the following to create your local database and user to allow access via php:
 
-### Database Setup
-Creating the database and import the schema:
+Import Setup scripts for database setup, user setup and data population.
 ```
+MariaDB[(none)]> source "/absolute/path/to/.../db/test_db_setup.sql"
 MariaDB[(none)]> source "/absolute/path/to/.../db/schema.sql"
+MariaDB[(none)]> source "/absolute/path/to/.../db/test_data.sql"
 ```
 
-User Setup:
-```
-MariaDB[ta_feedback]> CREATE USER 'myuser' IDENTIFIED BY 'mypassword';
-```
+#### Production setup with empty database
+
+1. Set environment variable specified in `db/config.php` for database connection.
```

Comment:
No. That variable might not be settable. Give explicit instructions.

Reply:
Do you think it's fine for production to just use the `t_tafs` setup just without any sample data?

Reply:
Provide instructions on how to import sample data. If it involves modifying a file, that's fine, but imagine that there is a new intern next summer and they are presented with this readme. How long will it take them to get the system working on their computer?